common/controllers/form-wizard.js

```diff
@@ -17,14 +18,37 @@ class FormController extends Controller {
     next()
   }
 
+  setupArbitraryMultipleFields(req, res, next) {
+    if (req.method === 'POST') {
```

Comment:
If this is the case, is this not something that should happen in the [`post`](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/7bfa5edb1af9f2b02cbe3ffcdc4b6e91ec9d9027/common/controllers/form-wizard.js#L55) method. This seems exactly it's purpose.

Reply:
It's simply too late to do this in the `post` middleware.

It 
a) needs to happen before form wizard proper gets its mitts on the fields
b) even more crucially needs to happen before setting up the multiple and conditional fields

Reply:
it's positioning is still the same but it no longer applies just to `POST`s